packages/rocketchat-channel-settings/client/views/channelSettings.js

```diff
@@ -219,10 +219,10 @@ Template.channelSettingsEditing.onCreated(function() {
 			isToggle: true,
 			processing: new ReactiveVar(false),
 			canView() {
-				return RocketChat.roomTypes.roomTypes[room.t].allowRoomSettingChange(room, RoomSettingsEnum.READ_ONLY);
+				return !room.broadcast && RocketChat.roomTypes.roomTypes[room.t].allowRoomSettingChange(room, RoomSettingsEnum.READ_ONLY);
```

Comment:
Can we please create a new room type of `broadcast` instead of polluting these methods. The room types api was intentionally changed for these style of changes.

Reply:
I will take a look. @graywolf336

Reply:
@graywolf336 I think for this case we need a new property instead of a new room type, because we thought cool having the ability to chose within a public or private broadcast room, so this way we're using all validations already implemented.